src/cmp.rs

```diff
@@ -14,6 +16,7 @@
 
 /// A type with an approximate equivalence relation.
 pub trait ApproxEq {
+    /// The epsilon type used measure an error.
     type Eps: Sized;
```

Comment:
Yes, we probably should (btw, the documentation link of `approx` is broken on crates.io and on the Readme). I'll create an issue.